### PR TITLE
Refactoring azure idp conf for new portal itf

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

This PR document how to configure IdP with Azure portal.

### Motivation

Our current [Azure as IdP instructions](https://docs.datadoghq.com/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp/) steps are through the old interface. 

### Preview link

* https://docs-staging.datadoghq.com/gus/azure-idp/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp/

### CODE REVIEW

File: content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md [hunk 1]
```diff
@@ -13,64 +13,75 @@ further_reading:
...
+    {{< img src="account_management/faq/non_gallery_application.png" alt="Non Gallery application" responsive="true" popup="true" style="width:20%;">}}
 
-6. Select "Add an application from the gallery"
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure.png" alt="Active_Directory_-_Microsoft_Azure" responsive="true" popup="true">}}
-    
+5. Name it (e.g **DatadogSSO_test**).
+
+6. Click on **Add**.
+
+7. Once the your application is successfully added, go in **Configure single sign-on (required)**.
```
Comment (cswatt) [inline content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md]: remove either "the" or "your"

File: content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md [hunk 2]
```diff
@@ -13,64 +13,75 @@ further_reading:
...
+
+8. Select the **SAML-based Sign-on** as Single Sign-on Mode.
+
+    {{< img src="account_management/faq/saml_based_sign_on.png" alt="Saml Based Sign on" responsive="true" popup="true" style="width:70%;">}}
+
+9. Navigate to the [Datadog SAML page][1], find the **Service Provider Entity ID** & **Assertion Consumer Service** URL on the right hand of the page.  Copy and paste those values in the **Identifier** and **Reply URL** text forms respectively:
+    In the datadog application:
```
Comment (cswatt) [inline content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md]: "In Datadog:"

File: content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md [hunk 3]
```diff
@@ -13,64 +13,75 @@ further_reading:
...
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure2.png" alt="Active_Directory_-_Microsoft_Azure2" responsive="true" popup="true">}}
+    In the Azure portal:
 
-8. Once the Application has been created, select "Configure single sign-on"
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure3.png" alt="Active_Directory_-_Microsoft_Azure3" responsive="true" popup="true">}}
+    {{< img src="account_management/faq/set_values_azure.png" alt="Set value azure" responsive="true" popup="true" style="width:70%;">}}
 
-9. Select "Microsoft Azure AD Single Sign-On"
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure4.png" alt="Active_Directory_-_Microsoft_Azure4" responsive="true" popup="true">}}
+11. Set `user.mail` as **User Identifier** Value:
```
Comment (cswatt) [inline content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md]: 11. Set `user.mail` as the value for **User Identifier**:

File: content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md [hunk 4]
```diff
@@ -13,64 +13,75 @@ further_reading:
...
 
-10. Navigate to https://app.datadoghq.com/saml/saml_setup, find the Service Provider Entity ID & Assertion Consumer Service URL on the right hand of the page.  Copy and paste those values in the "Identifier" and "Reply URL" text forms respectively
-    {{< img src="account_management/faq/Step10Redo.png" alt="Step10Redo" responsive="true" popup="true">}}
-    {{< img src="account_management/faq/Step10Redo2.png" alt="Step10Redo2" responsive="true" popup="true">}}
+    {{< img src="account_management/faq/user_identifier.png" alt="User Identifier" responsive="true" popup="true" style="width:70%;">}}
 
-11. On the next page Download Metadata (XML), check the box confirming you have configured SSO and press next. Use this file in step 17
+12. Enter your **Notification Email** at the bottom of the page for when the active signing certificate approaches its expiration date, notifications are then sent to this email address with instructions on how to update the certificate:
```
Comment (cswatt) [inline content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md]: 12. Enter your **Notification Email** at the bottom of the page. When the active signing certificate approaches its expiration date, notifications are sent to this email address with instructions on how to update the certificate:

File: content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md [hunk 5]
```diff
@@ -13,64 +13,75 @@ further_reading:
...
 
-12. Enter in an email address at which you would like to be notified about maintenance issues
+    {{< img src="account_management/faq/notification_email.png" alt="Notification email" responsive="true" popup="true" style="width:70%;">}}
 
-13. Once you are back to the applications main page, navigate to "Attributes"
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure6.png" alt="Active_Directory_-_Microsoft_Azure6" responsive="true" popup="true">}}
+13. Click at the bottom of the page on Step 5 **Configure DatadogSSO_test**.
```
Comment (cswatt) [inline content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md]: 13. Click at the bottom of the page on Step 5, **Configure DatadogSSO_test**.

File: content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md [hunk 6]
```diff
@@ -13,64 +13,75 @@ further_reading:
...
 
-14. In "SAML Token Attributes", hover over the line where "TYPE" is "user attribute (nameid)" (usually the first one) and click on the pencil icon to edit
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure7.png" alt="Active_Directory_-_Microsoft_Azure7" responsive="true" popup="true">}}
+14. Scroll down to the Step 3 of the *Configure DatadogSSO_test for single sign on* section, and download the **SAML XML Metadata** File.
```
Comment (cswatt) [inline content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md]: 14. Scroll down to Step 3 of the *Configure DatadogSSO_test for single sign on* section, and download the **SAML XML Metadata** File.

File: content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md [hunk 7]
```diff
@@ -13,64 +13,75 @@ further_reading:
...
 
-15. Change the attribute value to "user.mail" and press the check button
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure14.png" alt="Active_Directory_-_Microsoft_Azure14" responsive="true" popup="true">}}
+15. Go at the top of your **SSO Configuration section** and click **Save**.
```
Comment (cswatt) [inline content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md]: "Go to the top..."

File: content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md [hunk 8]
```diff
@@ -13,64 +13,75 @@ further_reading:
...
+
+16. Navigate back to [Datadog SAML page][1] and upload the **SAML XML Metadata** file downloaded in step 14:
```
Comment (cswatt) [inline content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md]: Capitalize "Step 14" for consistency.

File: content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md [hunk 9]
```diff
@@ -13,64 +13,75 @@ further_reading:
...
+
+    {{< img src="account_management/faq/SAML_Configuration___Datadog10.png" alt="SAML_Configuration___Datadog10" responsive="true" popup="true" style="width:70%;">}}
+
+17. Make sure to press the **Upload File** button after having chosen the XML file to upload.
+
+18. And that’s it! It should now say *SAML is ready* and that Valid IdP metadata installed.
```
Comment (cswatt) [inline content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md]: 18. And that’s it! It should now say *SAML is ready* and that Valid IdP metadata is installed.

File: content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md [hunk 10]
```diff
@@ -13,64 +13,75 @@ further_reading:
...
 
-16. Press "Apply Changes" at the bottom of the screen
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure9.png" alt="Active_Directory_-_Microsoft_Azure9" responsive="true" popup="true">}}
 
-17. Navigate back to https://app.datadoghq.com/saml/saml_setup and upload the xml file downloaded in step 11:
-    {{< img src="account_management/faq/SAML_Configuration___Datadog10.png" alt="SAML_Configuration___Datadog10" responsive="true" popup="true">}}
+#### Optional 
 
-18. Make sure to press "Upload File" after choosing the XML file
+If you are using a SSO via a Datadog button or link, you need to add a Sign-on URL. To do this, navigate back to the configure SSO Configuration section of the Azure Application (Step 8) check off **Show advanced URL settings**:
```
Comment (cswatt) [inline content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md]: If you are using an SSO via a Datadog button or link, you need to add a Sign-on URL. To do this, navigate back to the SSO Configuration section of the Azure Application (Step 8) and check off **Show advanced URL settings**:

File: content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md [hunk 11]
```diff
@@ -13,64 +13,75 @@ further_reading:
...
 **Note**: an Azure AD Premium Subscription is required to set this up
 
-1. Navigate to `manage.windowsazure.com`
+#### Configuration 
 
-2. Go to the Active Directory sub menu
+1. Navigate to `https://portal.azure.com/`.
 
-3. Select the active directory you wish to use for SSO
+2. Navigate to the **Enterprise applications** service.
```
Comment (DanielLanger) [inline content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md]: I think there should be a step before this to navigate to the Azure Active Directory tab on the left hand menu, no?

File: content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md [hunk 12]
```diff
@@ -13,64 +13,75 @@ further_reading:
...
+    {{< img src="account_management/faq/set_values_azure.png" alt="Set value azure" responsive="true" popup="true" style="width:70%;">}}
 
-9. Select "Microsoft Azure AD Single Sign-On"
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure4.png" alt="Active_Directory_-_Microsoft_Azure4" responsive="true" popup="true">}}
+11. Set `user.mail` as **User Identifier** Value:
 
-10. Navigate to https://app.datadoghq.com/saml/saml_setup, find the Service Provider Entity ID & Assertion Consumer Service URL on the right hand of the page.  Copy and paste those values in the "Identifier" and "Reply URL" text forms respectively
-    {{< img src="account_management/faq/Step10Redo.png" alt="Step10Redo" responsive="true" popup="true">}}
-    {{< img src="account_management/faq/Step10Redo2.png" alt="Step10Redo2" responsive="true" popup="true">}}
+    {{< img src="account_management/faq/user_identifier.png" alt="User Identifier" responsive="true" popup="true" style="width:70%;">}}
```
Comment (DanielLanger) [inline content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md]: This image has a a funny typo in the email. Spelled company compagny :) not a big deal but might want to fix

**Pull request merged**